client/my-sites/subscribers/components/subscriber-data-views/subscriber-data-views.tsx

```diff
@@ -345,24 +346,33 @@ const SubscriberDataViews = ( {
 				{ shouldShowLaunchpad ? (
 					<EmptyComponent />
 				) : (
-					<DataViews< Subscriber >
-						data={ data }
-						fields={ fields }
-						view={ currentView }
-						onClickItem={ handleSubscriberOnClick }
-						onChangeView={ setCurrentView }
-						selection={
-							selectedSubscriber ? [ selectedSubscriber.subscription_id.toString() ] : undefined
-						}
-						onChangeSelection={ handleSubscriberSelect }
-						isLoading={ isLoading }
-						paginationInfo={ paginationInfo }
-						getItemId={ ( item: Subscriber ) => item.subscription_id.toString() }
-						defaultLayouts={ selectedSubscriber ? { list: {} } : { table: {} } }
-						actions={ actions }
-						search
-						searchLabel={ translate( 'Search by name, username or email…' ) }
-					/>
+					<>
+						<SubscriberTotals
+							totalSubscribers={ grandTotal }
+							filteredCount={ subscribers.length }
```

Comment:
I realized `subscribers.length` is giving us the number on that page. I'll look for a fix.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/99177/commits/2929f80293fb7a86b45b4aff01838700128f483a